examples/dualitygap.jl

```diff
@@ -0,0 +1,44 @@
+
+# Example of polyhedral OA failure due to infinite duality gap from "Polyhedral approximation in mixed-integer convex optimization - Lubin et al 2016"
```

Comment:
also for now these are mostly just to make it easy for us to run problems and observe output to debug